core/reactor_common.c

```diff
@@ -1978,13 +1983,20 @@ void initialize(void) {
     current_tag.time = physical_start_time;
     start_time = current_tag.time;
 
+    #ifdef BIT_32
+    LF_PRINT_DEBUG("Start time: %ldns", start_time);
```

Comment:
Should this be `%ldus`? If the time units are microseconds.

Reply:
Added nested options for what to print.

Reply:
@arengarajan99: I didn't understand your comment above.

Reply:
Oh, by "nested options" you must have meant conditionals...